stock_production_lot_expiry/models/stock_lot.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+
+from odoo import _, api, fields, models
+from odoo.osv.expression import NEGATIVE_TERM_OPERATORS
+
+
+class StockProductionLot(models.Model):
+
+    _inherit = "stock.lot"
+
+    is_expired = fields.Boolean(
```

Comment:
@jbaudoux @glitchov  Maybe we could use the `product_expiry_alert`core field as that one is used in some flows, for instance there : https://github.com/odoo/odoo/blob/16.0/addons/product_expiry/models/stock_move_line.py#L16